authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/integration/OTPManager.java

```diff
@@ -135,12 +132,17 @@ private String generateOTP(String uin) throws IdAuthUncheckedException {
 			RestRequestDTO restRequest = restRequestFactory.buildRequest(RestServicesConstants.OTP_GENERATE_SERVICE,
 					reqWrapper, ResponseWrapper.class);
 			ResponseWrapper<Map<String, String>> response = restHelper.requestSync(restRequest);
-			if (response != null && response.getResponse().get("status").equals(USER_BLOCKED)) {
+			if ( response!=null && response.getResponse()!=null && response.getResponse().get("status").equals(USER_BLOCKED)) {

```

Comment:
add a null check for `response.getResponse().get("status")` also.

Reply:
Added check for response.getResponse().get("status") also.